Windows/Debugger/Debugger_MemoryDlg.h

```diff
@@ -6,19 +6,29 @@
 #include "Core/MemMap.h"
 
 #include "Core/Debugger/DebugInterface.h"
-
+#include "CtrlMemView.h"
 #include "Common/CommonWindows.h"
 
 class CMemoryDlg : public Dialog
 {
 private:
 	DebugInterface *cpu;
 	static RECT slRect;
-
+	RECT winRect, srRect;
+	CtrlMemView *memView;
+	HWND memViewHdl , symListHdl, editWnd, searchBoxHdl, srcListHdl;
 	BOOL DlgProc(UINT message, WPARAM wParam, LPARAM lParam);
+	
+	struct SearchResult {
+		u32 address;
+		const char* string;
+	};
+
 public:
 	int index; //helper 
 
+	void searchBoxRedraw(std::vector<u32>);
```

Comment:
we always name parameters, even in headers.

Reply:
ok